server/src/packages/packages.controller.ts

```diff
@@ -54,6 +54,47 @@ export const PASFORM_PROJECTADDRESS_ATTRS = [
   'statecode',
 ]
 
+export const RWCDSFORM_ATTRS = [
```

Comment:
Should probably be a space `RWCDS_FORM_ATTRS` because that's how `PAS_FORM` is named